README.md

```diff
@@ -47,7 +47,7 @@ For everyone else, the detailed instructions to install Alacritty can be found
 
 ### Requirements
 
-- OpenGL 3.3 or higher
+- OpenGL ES 2.0 or higher
```

Comment:
I'm not sure how to adjust it, since I have a feeling that OpenGL 2.1 device without gles2 won't work, since shaders are gles2. Though it's a matter of adjusting syntax in shaders.

Reply:
Should be almost irrelevant at that point. But `GLES2 support` might suffice?